lib/lorekeeper/json_logger.rb

```diff
@@ -96,7 +96,16 @@ def inspect
     end
 
     def write(message)
-      super(Oj.dump(message, mode: :compat, cache_keys: true, cache_str: 5) << "\n")
+      json_message =
+        begin
+          Oj.dump(message, mode: :compat, cache_keys: true, cache_str: 5)
+        rescue JSON::GeneratorError
+          Oj.dump(message)
+        rescue => e
+          Oj.dump({ MESSAGE: e.message })
```

Comment:
`MESSAGE` is a constant 🐶 
```suggestion
          Oj.dump({ MESSAGE => e.message })
```

Reply:
we are almost there!! one last thing! 😃